docs/examples/05-job_schedule.jl

```diff
@@ -0,0 +1,133 @@
+# This example demonstrates a job scheduling through adding the
+# number 100 to every component of the vector data. The master
+# assigns one element to each slave to compute the operation.
+# When the worker is finished, the master sends another element
+# until each element is added 100
+# Inspired on
+# https://www.hpc.ntnu.no/ntnu-hpc-group/vilje/user-guide/software/mpi-and-mpi-io-training-tutorial/basic-mpi/job-queue
+
+using MPI
+
+function job_queue(data,f)
+    MPI.Init()
+
+    comm = MPI.COMM_WORLD
+    rank = MPI.Comm_rank(comm)
+    world_size = MPI.Comm_size(comm)
+    workers = world_size - 1
```

Comment:
```suggestion
    nworkers = world_size - 1
```
since it is the number of workers, not the set of workers.